### PR TITLE
Don't copy systemvm.iso files to agent if >=4.16

### PR DESCRIPTION
**Pull request by itsayushpandey**
Skip copying iso files for 4.16+ Apache Cloudstack version from local folder.

Fixes: #33

### CODE REVIEW

**Comment by itsayushpandey** [comment]
@nvazquez , @rohityadavcloud - had this minor fix on Monkeybox left. 

I had encountered some other issues in my internship earlier in May 2023 as it was outdated beyond 4.15, but seems like all that has been in the latest main branch in August. Thank you!

**Review by rohityadavcloud — APPROVED** [review]
LGTM - thanks for fixing

File: mbx [hunk 1]
```diff
@@ -625,8 +625,11 @@ agentscp() {
   issh root@$kvm_host "mkdir -p /etc/cloudstack/agent/ /usr/share/cloudstack-common/{lib,scripts,vms} /usr/share/cloudstack-agent/{lib,plugins} /var/log/cloudstack/agent/"
   issh root@$kvm_host "echo 'paths.pid=/var/run' > /etc/cloudstack/agent/environment.properties; echo 'paths.script=/usr/share/cloudstack-common' >> /etc/cloudstack/agent/environment.properties"
 
-  echo "[acs agent] Copied systemvm.iso"
-  iscp $CUR/systemvm/dist/systemvm.iso root@$kvm_host:/usr/share/cloudstack-common/vms/
+  local_version=$(ls $CUR/client/target/cloud-client* | sed 's/.*version="//g' | sed 's/".*//g' | sed 's/-SNAPSHOT//g' | grep -Po '4.[\d]*')
```
Comment (rohityadavcloud) [inline mbx:628]: I think we can safely remove this, as 4.15 and older were really old and EOL'd releases; we don't expect any development work on them.

Comment (rohityadavcloud) [inline mbx:628]:
```suggestion
  local_version=$(ls $CUR/client/target/cloud-client* | sed 's/.*version="//g' | sed 's/".*//g' | sed 's/-SNAPSHOT//g' | grep -Po '4.[\d]*')
```

File: mbx [hunk 2]
```diff
@@ -625,8 +625,11 @@ agentscp() {
...
+  if [[ $(echo $local_version | sed 's/4.//g') -lt 16 ]]; then
+  	echo "[acs agent] Copied systemvm.iso"
+  	iscp $CUR/systemvm/dist/systemvm.iso root@$kvm_host:/usr/share/cloudstack-common/vms/
+  fi
```
Comment (rohityadavcloud) [inline mbx:633]:
```suggestion

```

**Pull request merged**